helm/robusta/values.yaml

```diff
@@ -443,6 +443,7 @@ kubewatch:
     limits:
       cpu: ~
   additional_env_vars: []
+  priorityClassName: default
```

Comment:
Can you change the default value to "" ?
So we don't change the existing behavior